.github/workflows/pull_request.yml

```diff
@@ -136,9 +141,11 @@ jobs:
         run: |
           course_name=$(head -n 1 _website.yml | cut -d'"' -f 2| tr " " "-")
```

Comment:
```suggestion
```
appears not to be used. Did you mean to echo it to the log?

Reply:
Deleted!